streamer/src/quic.rs

```diff
@@ -160,6 +160,7 @@ pub struct StreamStats {
     pub(crate) connection_add_failed: AtomicUsize,
     pub(crate) connection_add_failed_unstaked_node: AtomicUsize,
     pub(crate) connection_setup_timeout: AtomicUsize,
+    pub(crate) connection_setup_error: AtomicUsize,
```

Comment:
i'm not sure an aggregate of setup error counts is going to be valuable.  we probably need individual counters